packages/framework/src/Model/GoPay/GoPayClientFactory.php

```diff
@@ -4,43 +4,102 @@
 
 namespace Shopsys\FrameworkBundle\Model\GoPay;
 
+use GoPay\Definition\TokenScope;
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
 use Shopsys\FrameworkBundle\Model\GoPay\Exception\GoPayNotConfiguredException;
+use Shopsys\FrameworkBundle\Model\GoPay\Exception\GoPayNotEnabledOnDomainException;
 
 class GoPayClientFactory
 {
+    protected const string PRODUCTION_URL = 'https://gate.gopay.cz/';
+    protected const string TEST_URL = 'https://gw.sandbox.gopay.com/';
+
+    /**
+     * @var array<int, \Shopsys\FrameworkBundle\Model\GoPay\GoPayClient>
+     */
+    protected array $clientByDomainId = [];
```

Comment:
:information_source: 
I saw few more constants with singular name, that are typed as array. I ignored it, because I was not sure, if it is on purpose

Reply:
changed in 9657622c83.

I think the others are fine, if you have any examples, please let me know